index.html

```diff
@@ -693,9 +693,22 @@
               </li>
               <li>Determine which <a>payment handlers</a> support any of the
               <a>payment method identifiers</a> given <var>identifiers</var>.
-              For each resulting payment handler, if payment method specific
-              capabilities supplied by the payment handler match those provided
-              by <var>data</var>, the payment handler matches.
+              </li>
+              <li>For each <var>handler</var> of the resulting payment
+              handlers:
+                <ol>
+                  <li>If required by the specification that defines the
+                  <var>handler</var>, <a data-cite=
+                  "!WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
```

Comment:
I guess if we go with this architecture you need to JSON-parse before you convert, otherwise you're just converting it to an IDL DOMString, which is no good.